pandas/io/formats/format.py

```diff
@@ -575,6 +575,16 @@ def __init__(
                     " DataFrame number of columns({dlen})"
                 ).format(flen=len(formatters), dlen=len(frame.columns))
             )
+
+        if isinstance(col_space, list) and len(col_space) != len(frame.columns):
+            raise ValueError(
+                (
+                    "Length of col_space list ({clen}) should match"
+                    " number of DataFrame columns ({dlen})".format(
+                        clen=len(col_space), dlen=len(frame.columns)
+                    )
+                )
+            )
```

Comment:
We'll be dropping Python 3.5 in `1.0`, so let's use F-string syntax

Reply:
Hi, done. Does the message seem alright?